Week1/Mohammad-Darwesh-JS2-week1/app.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+{
+  const bookTitles = [
+    'the_great_gatsby',
+    'city_on_the_edge',
+    'harry_potter',
+    'james_and_the_giant_peach',
+    'set_me_free',
+    'a_blind_eye',
+    'good_to_great',
+    'a_room_away_from_the_wolves',
+    'church_of_marvels',
+    'after_the_rain',
+  ];
+  const booksCovers = {
+    the_great_gatsby: 'http://flavorwire.files.wordpress.com/2012/08/gatsby.jpg',
+    city_on_the_edge: 'https://images.penguinrandomhouse.com/cover/9780425284469',
+    harry_potter: 'https://hpmedia.bloomsbury.com/rep/s/9781408855911_309576.jpeg',
+    james_and_the_giant_peach: 'https://images-na.ssl-images-amazon.com/images/I/51A5cHQhSqL.jpg',
+    set_me_free:
+      'https://dwtr67e3ikfml.cloudfront.net/bookCovers/dbce49e06c7ed4ff54b52d21ebcb41dd38ecb698',
+    a_blind_eye: 'https://images-eu.ssl-images-amazon.com/images/I/51SN7zgwP7L.jpg',
+    good_to_great: 'https://pictures.abebooks.com/HENNIKERBOOKFARM/22445761977.jpg',
+    a_room_away_from_the_wolves:
+      'https://images-na.ssl-images-amazon.com/images/I/5116zAISjzL._SX329_BO1,204,203,200_.jpg',
+    church_of_marvels:
+      'https://i.pinimg.com/originals/b8/19/0d/b8190d0fb7bcb1ced1bc4d69c3637c22.jpg',
+    after_the_rain:
+      'https://vicsmediaroom.files.wordpress.com/2016/04/after-the-rain-book-cover.jpg?w=308',
+  };
+  const div = document.createElement('div');
+  document.body.appendChild(div);
+  const header = document.createElement('h1');
+  const addHeader = document.body.getElementsByTagName('div')[0];
+  addHeader.appendChild(header);
+  header.innerHTML = 'My favorite books';
+  const list = document.createElement('ul');
+  div.appendChild(list);
+  list.setAttribute('class', 'books_list');
+  function makingList() {
+    for (let item of bookTitles) {
+      const booksList = document.createElement('li');
+      list.appendChild(booksList);
+      booksList.setAttribute('id', item);
```

Comment:
You have a function later that inserts the cover (as should happen from the spec) why are you adding it here as well?

Reply:
Hello Rohni ,
Thank you for your  feedback and sure i will consider your useful advises.
about insert the cover ......my idea from tow function that one for HTML structures(elements,tags,attributes) and the second one  for insert stuff in the elements.